onnxruntime/core/framework/debug_node_inputs_outputs_utils.h

```diff
@@ -30,6 +30,8 @@ constexpr const char* kNameFilter = "ORT_DEBUG_NODE_IO_NAME_FILTER";
 constexpr const char* kOpTypeFilter = "ORT_DEBUG_NODE_IO_OP_TYPE_FILTER";
 // set to non-zero to dump data to files instead of stdout
 constexpr const char* kDumpDataToFiles = "ORT_DEBUG_NODE_IO_DUMP_DATA_TO_FILES";
+// set to non-zero to append rank to filename
+constexpr const char* kRankToFileName = "ORT_DEBUG_NODE_IO_RANK_TO_FILE_NAME";
```

Comment:
nit: how about ORT_DEBUG_NODE_IO_APPEND_RANK_TO_FILE_NAME?
also "rank" by itself is a bit vague, maybe in the comment mention that it is the openmpi world rank?